contributor-docs/code-change-guide.md

```diff
@@ -115,6 +115,8 @@ To run a Gradle task, use the command `./gradlew -p <PROJECT_PATH> <TASK>` or th
 ./gradlew :sdks:java:harness:test
 ```
 
+**It is recommended to run `./gradlew clean` if you run into some strange errors such as `java.lang.NoClassDefFoundError`.**
```

Comment:
The main text is for user to follow steps. When there are issue happens its hard to find solution if they are scattered in the doc. How about having a "Common Issue" sections before Appendix?

Reply:
Good idea. Moved this to a new `Common Issues`